src/main/java/com/github/fabienrenaud/jjb/databind/Deserialization.java

```diff
@@ -15,6 +17,16 @@
  * @author Fabien Renaud
  */
 public class Deserialization extends JsonBench {
+    static final ObjectReaderProvider featuresProvider = new ObjectReaderProvider();
+    static final JSONReader.Context featuresContext;
+    static {
+        featuresProvider.setDisableArrayMapping(true);
+        featuresProvider.setDisableAutoType(true);
+        featuresProvider.setDisableJSONB(true);
+        featuresProvider.setDisableReferenceDetect(true);
+        featuresProvider.setDisableSmartMatch(true);
+        featuresContext = new JSONReader.Context(featuresProvider);
+    }
```

Comment:
Please hide all this behind `JsonProvider`. Introduce a fastjson() and fastjson_features() provider there.